etcdserver/api/v3rpc/interceptor.go

```diff
@@ -48,6 +48,11 @@ func newUnaryInterceptor(s *etcdserver.EtcdServer) grpc.UnaryServerInterceptor {
 			return nil, rpctypes.ErrGRPCNotCapable
 		}
 
+		// TODO: add test in clientv3/integration to verify behavior
```

Comment:
+1, ensuring at least some of the main operations are rejected or accepted appropriately sounds valuable.